sql/src/main/java/io/crate/analyze/DropBlobTableAnalyzedStatement.java

```diff
@@ -33,4 +37,17 @@ public DropBlobTableAnalyzedStatement(Schemas schemas, boolean ignoreNonExistent
     public <C, R> R accept(AnalyzedStatementVisitor<C, R> analyzedStatementVisitor, C context) {
         return analyzedStatementVisitor.visitDropBlobTableStatement(this, context);
     }
+
+    public void table(TableIdent tableIdent) {
+        try {
+            tableInfo = (BlobTableInfo) schemas.getTableInfo(tableIdent);
```

Comment:
I think the method could be kept in the Abstract class and just cast to the generic?
`tableInfo = (T) schemas.getTableInfo(tableIdent)`